docs/development.rst

```diff
@@ -42,3 +42,25 @@ To only run a specific test from that directory, for example
 .. code-block:: shell
 
   pytest -k multi_line-class_docstring
+
+
+Primer
+-------
+
+To check if any changes create any unforeseen regressions all pull requests are tested
+with our primer. This process is heavily inspired on other open source projects and our
+own primer is based on the one used by `mypy <http://mypy-lang.org>`_.
+
+You can also run the primer locally with minimal set-up. First you will need to make
+a duplicate of your ``pydocstringformatter`` directory in a new ``./program_to_test``
+directory. This is required so that the primer can check out multiple versions of the
+program and compare the difference.
+
+The next step is to follow the bash script in the ``Run primer`` step in the ``primer``
+workflow file, found at ``.github/workflows/primer.yaml``. This should allow you to run
+all necessary steps locally.
```

Comment:
Maybe we could create a test with pytest for that ? It seem automatable even if not very convenient because you need to duplicate the git repository.

Reply:
Yeah, I think it does not offer much: when would the test pass or fail?
At some point I could explore trying to create a script that does the same and handles cloning the repository, but for now I think this is fine.

Reply:
Ha right it makes no sense as a test. A script to set it up is more like it.